apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -173,6 +181,11 @@ const FapProposalsAndAssignmentsTable = ({
     settingsFormatToUse: SettingsId.DATE_FORMAT,
   });
   const { enqueueSnackbar } = useSnackbar();
+  const { featuresMap } = useContext(FeatureContext);
+
+  const useWarning = featuresMap.get(
```

Comment:
```suggestion
  const isUseConflictOfInterestWarningEnabled = featuresMap.get(
```

Reply:
Just to get the variable naming more aligned with the feature flag name, so scanning the code is more easier.